pdns/recursordist/docs/upgrade.rst

```diff
@@ -7,9 +7,20 @@ When upgrading several versions, please read **all** notes applying to the upgra
 4.6.x to master
 ---------------
 
+Zone to Cache Changes
+^^^^^^^^^^^^^^^^^^^^^
+The :ref:`ztc` feature now validates ``ZONEMD`` records. This means that zones containing invalid ``ZONEMD`` record will
+be rejected by default, while previously the ``ZONEMD`` records would be ignored. For more detail, refer to :ref:`ztc`.
+
+Asynchronous retrieval of ``AAAA`` records for nameservers
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+If IPv6 is enabled in for outgoing queries using :ref:`setting-query-local-address` the Recursor will schedule an asynchronous task to resolve IPv6 addresses of nameservers it did not learn otherwise.
```

Comment:
```suggestion
If IPv6 is enabled for outgoing queries using :ref:`setting-query-local-address`, the Recursor will schedule an asynchronous task to resolve IPv6 addresses of nameservers it did not otherwise learn.
```